runner/tasks.py

```diff
@@ -14,37 +14,88 @@
 logger = logging.getLogger(__name__)
 
 
+def create_jobs_from_operator(operator):
+    jobs = operator.get_jobs()
+
+    valid_jobs, invalid_jobs = [], []
+    for job in jobs:
+        valid_jobs.append(job) if job[0].is_valid() else invalid_jobs.append(job)
+
+    operator_run = OperatorRun.objects.create(operator=operator, total_num_runs=len(valid_jobs))
+    Run.objects.bulk_update(valid_jobs, operator_run_id=operator_run.id)
+
+    for job in valid_jobs:
+        logger.info("Creating Run object")
+        run = job[0].save()
+        logger.info("Run object created with id: %s" % str(run.id))
+        create_run_task.delay(str(run.id), job[1], None)
+
+    for job in invalid_jobs:
+        logger.error("Job invalid: %s" % str(job[0].errors))
```

Comment:
we don't associate invalid runs with an operator run since it would break the conditions -- let me know if this is incorrect.

Reply:
We might want to report invalid runs somehow. Currently we are just ignoring them. Maybe the error table which we mentioned makes sense for this?

Reply:
That's something we can do, or we can explore something like `logz.io` or even Sentry.